homework/hackyourtemperature/app.js

```diff
@@ -30,9 +30,10 @@ app.post("/weather/:cityName", async (req, res) => {
     } else if (weather.cod === 400) {
       return res.status(400).json({ weatherText: weather.message });
     } else {
+      const tempCelsius=weather.main.temp-−273.15
       return res.status(200).json({
         weatherText: weather.name,
-        temperature: weather.main.temp.toFixed(2),
+        temperature: tempCelsius,
```

Comment:
Making a variable here is a nice idea, it makes the code easier to read.

When doing it, the `toFixed(2)` got lost in the process

```suggestion
        temperature: tempCelsius.toFixed(2),
```